boards/DCM/Inc/App/configs/App_TorqueRequestThresholds.h

```diff
@@ -7,3 +7,7 @@
  * @see Setting up the Controller to run with Emrax Motor Rev 1.2 PDF pg. 6
  */
 #define MAX_TORQUE_REQUEST_NM 132.0f;
+
+// For torque plausibility checking between FSM and DCM
+#define TORQUE_REQUEST_UPPER_BOUND_FACTOR 1.1f
+#define TORQUE_REQUEST_LOWER_BOUND_FACTOR 0.9f
```

Comment:
can place this in the App_SetPeriodicCanSignals.c file for the FSM

Reply:
make sure you put stuff in braces as well i.e. `#define YEAH (1.1f)`

Reply:
Done